src/Learning/KWDataPreparation/KWClassStats.cpp

```diff
@@ -887,31 +892,39 @@ void KWClassStats::WriteReport(ostream& ost)
 	if (GetTargetAttributeName() != "")
 	{
 		// Attribut cible
-		// On demande un affichage complet (source et cible) pour forcer
-		// l'utilisation explicite du libelle "Target"
-		assert(GetTargetValueStats()->GetSourceAttributeNumber() == 0);
 		ost << "\n";
-		GetTargetValueStats()->WriteAttributeArrayLineReports(ost, true, true);
+		ost << "Target variable"
+		    << "\t" << KWType::ToString(GetTargetAttributeType()) << "\t"
+		    << TSV::Export(GetTargetAttributeName()) << "\n";
 
-		// Statistiques descriptives
-		if (GetTargetAttributeType() == KWType::Continuous or
-		    (GetTargetAttributeType() == KWType::Symbol and
-		     GetTargetDescriptiveStats()->GetValueNumber() > GetTargetValueLargeNumber(GetInstanceNumber())))
-		{
-			ost << "\n";
-			GetTargetDescriptiveStats()->WriteReport(ost);
-		}
+		// Modalite cible principale
+		if (GetTargetAttributeType() == KWType::Symbol and GetMainTargetModality() != Symbol())
```

Comment:
I'm not sure I get this test:
- target attribute type == `KWType::Symbol` and 
- main target modality != `Symbol()`
what could the main target modality be for Symbol target attribute types?

Reply:
Symbol est le type interne du code pour gérer le type Categorical
MinTargetModality est le nom interne pour "Main target value"
Ici, on test juste que dans la cas d'une classification supervisée (variable cible de type Categorical), la valeur cible principale est non vide (non spécifiée par l'utilisateur)

Reply:
OK, then I would update the preceding comment to something like:

```c++
// Modalite cible principale non-vide (non-specifiee par l'utilisateur)
```

Reply:
Fait, avec le commentaire suivant
`// Modalite cible principale si specifiee par l'utilisateur`

Reply:
Ok, much clearer, thank you.